perfmetrics/scripts/continuous_test/gcp_ubuntu/periodic_experiments/README.md

```diff
@@ -0,0 +1,57 @@
+## Copyright 2023 Google Inc. All Rights Reserved.
+##
+## Licensed under the Apache License, Version 2.0 (the "License");
+## you may not use this file except in compliance with the License.
+## You may obtain a copy of the License at
+##
+##     http:#www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+
+## Experiment Configuration JSON File README
+## This JSON file contains configurations for different experiments.
+
+## Configuration Format:
+```
+{
+    "experiment_configuration": [
+        {
+            "config_name": "<config_name>",
+            "gcsfuse_flags": "<gcsfuse_flags>",
+            "branch": "<branch>",
+            "end_date": "<end_date>"
+        },
+        {
+        ...
+        },
+        ...
+    ]
+}
+```
+## Key Descriptions: 
+1. config_name (string): The name of the configuration. This should be a unique identifier for each experiment.
+2. gcsfuse_flags (string): Flags to be passed to the gcsfuse command when running the experiment. 
+3. branch (string): The Git branch to use for the experiment.  
+4. end_date (string): The experiment will run every day till this date and time. Format: "YYYY-MM-DD HH:MM:SS".
+
+## Configuration Example
+### An example of the configuration object in the experiment_configuration array:
+```
+"config_name": "TestConfiguration1"
+"gcsfuse_flags": "--implicit-dirs --max-conns-per-host 100 --enable-storage-client-library --debug_fuse --debug_gcs --stackdriver-export-interval=30s"
+"branch": "master"
+"end_date": "2023-12-30 05:30:00"
+```
+
+## Uniquely Identifying an Experiment
```

Comment:
let's keep it simple - just say a configuration name uniquely defines an experiment and two experiments can't have same configuration name. Also, once a configuration is defined, you can't edit the gcsfuse flags and branch. if you want to change them then add new experiment configuration.

Also, 
1. just filter based on configuration name here: https://github.com/GoogleCloudPlatform/gcsfuse/blob/ab6948d3cdc2f95053982baaab246be45e66a69e/perfmetrics/scripts/bigquery/experiments_gcsfuse_bq.py#L263.
2. And if you find only 1 row from /1 then check both gcsfuse flags and branch should be same as the passed values. If not then raise exception

Reply:
For the changes suggested in experiments_gcsfuse_bq.py: The current query is doing the work of both the suggested query's breakdown combined. Is the breakdown to raise precise exception messages?